DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/ScoresTable.Script.txt

```diff
@@ -37,11 +39,13 @@ Text GetManialink() {
 		<frameinstance modelid="playerstatistic-card" pos="43" id="playerstatistic-assists"/>
 
 		<!-- Data -->
-		<quad pos="-50" z-index="1" size="10 10" bgcolor="FFF" halign="left" valign="center" id="avatar"/>
-		<label pos="-37.5 0" z-index="0" size="30 10" text="Player Name" textcolor="{{{ CL::RgbToHex6(Colors::C_NeutralLight) }}}" halign="left" valign="center2" textfont="GameFontRegular" id="name" textsize="2.5"/>
+		<quad pos="-49 0" z-index="1" size="10 10" bgcolor="FFF" halign="left" valign="center" id="avatar" keepratio="Fit"/>
```

Comment:
It now has 1 unit distance to the left, but in case of an avatar that takes full height, it will still hit the top and bottom edge. maybe the size should be changed to "8 8", so we always have at least a unit of distance.